apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -14,48 +16,59 @@ import {
   ONE_TIME_DIVIDEND,
   RECURRING_DIVIDEND,
   ONE_TIME_MERIT,
+  READY,
+  CLAIMED,
 } from '../../utils/constants'
 import { Empty } from '../Card'
 import Metrics from './Metrics'
-import { useAppState } from '@aragon/api-react'
+import { useAppState, useNetwork } from '@aragon/api-react'
 import BigNumber from 'bignumber.js'
-import { displayCurrency, getStatus } from '../../utils/helpers'
+import { displayCurrency, getStatus, getStatusId } from '../../utils/helpers'
 
 const MyRewards = ({
   myRewards,
   myMetrics,
   viewReward,
   claimReward,
+  claimHashes,
 }) => {
+  const network = useNetwork()
   const rewardsEmpty = myRewards.length === 0
 
+  const getEtherscanLink = reward => {
```

Comment:
Does this use any component state? If not, let's move it outside the definition of `MyRewards`.

Reply:
It does use `claimHashes` which is a prop. And also `network`, which is a hook. So not possible.

Reply:
I'd prefer we use `useCallback`, in that case.

```js
const getEtherescanLink = useCallback(reward => {
  // existing function here
}, [ claimHashes, network ])
```

Benefits:

1. The function will only be redefined if `claimHashes` or `network` change, rather than being redefined on every render of this function
2. Code clarity/communication: this helps anyone else looking at this code quickly understand what bits of component state this function relies on (`claimHashes` and `network`)